### DISCUSSION TITLE
1.21!!!!!

### DISCUSSION THREAD

**Discussion by Fengyuam** [maruohon/litematica#870] [General]
mc1.21!!!!!

**Comment by Samalando** [comment]
Masa has stated that he will not continue to update past 1.20.5 due to minor versions getting mod-breaking changes and he also has burnout. You can get 1.21 from [sakura ryokos github](https://github.com/sakura-ryoko), and that will be where the mods will get updated until further notice. I do think that it is on curseforge though.
